### PR TITLE
Adds blower bypass

### PR DESCRIPTION
**Pull request by gracecanfield**
https://respiraworks.workast.io/list/5ec1ec3d3425e910786b4ab2?tasks

* Adds blower bypass page
* Updates pizza build index
* Updates alpha build index

### CODE REVIEW

File: 4_Manufacturing_Design/Alpha_Build_Instructions/pizza_build.md [hunk 1]
```diff
@@ -137,8 +139,8 @@ You may need anywhere between 1 and 3 such pinch valves, depending on the comple
 * **Reduced compliance test lung:** To test basic function a ventilator check lung or a breathing bag is sufficient,
 but can really only work up to 10cmH2O. To be able to test higher PIP values, you need to reduce the compliance of the
 lung so you don’t just inflate it like a balloon (or pop it). View more details on reduced compliance test lung [here](pizza_compliance_test_lung).
+* **Blower bypass:** The purpose of the blower bypass is to prevent strain on blower when the inhale pinch valve is closed. The sub-assembly for this component is documented [here](pizza_blower_bypass).
 * **TODO: check valve**
```
Comment (martukas) [inline 4_Manufacturing_Design/Alpha_Build_Instructions/pizza_build.md:143]: Also add a TODO for oxygen supply limb

File: 4_Manufacturing_Design/Alpha_Build_Instructions/pizza_blower_bypass/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+[4mcmc]:   https://www.mcmaster.com/4880K154
+[5mcmc]:   https://www.mcmaster.com/5372K128
+[6mcmc]:   https://www.mcmaster.com/53415K204
+[7mcmc]:   https://www.mcmaster.com/4880K343
+
+## Assembly
+
+Parts:
+
+![Parts](bypass-parts.jpg)
```
Comment (martukas) [inline 4_Manufacturing_Design/Alpha_Build_Instructions/pizza_blower_bypass/README.md:43]: Where you have 6 in the picture is actually the alternative config with 7 and 5. I have not received #6 from mcmaster yet. Will provide additional photo when I have it.

File: 4_Manufacturing_Design/Alpha_Build_Instructions/Blower-Bypass.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+| ------ |---------:| ------------- | ------------------- | ------------:|:---------:|-----------------| ----- |
+| 1      |          | McMaster-Carr | 5894K34           | 2.43 / ft   | buy       | [C][1mcmc]      | Continuous-Flex Soft Tygon PVC Tubing |
+| 2      |          | DERNORD | 40141607           | 12.79   | buy       | [C][1mcmc]      | Ball valve, as adjustable orifice |
+| 3      |          | McMaster-Carr | 5372K112           | 4.81/10   | buy       | [C][3mcmc]      | To connect ball valve to tubing |
+| 3      |          | McMaster-Carr | 5372K112           | 4.81/10   | buy       | [C][3mcmc]      | To connect ball valve to tubing |
+
+
+[1mcmc]   [https://www.mcmaster.com/5894K34 |
+[2z]   [https://www.amazon.com/gp/product/B07BSQ5X8H|
+3mcmc[https://www.mcmaster.com/5372K112|
```
[the remark this answers is not in the document]
Comment (gracecanfield) [reply 4_Manufacturing_Design/Alpha_Build_Instructions/Blower-Bypass.md]: Yes - I don't know how this got messed up. Deleted the markdown file in the Alpha_Build_Instructions folder.

File: 4_Manufacturing_Design/Alpha_Build_Instructions/Blower-Bypass.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Blower Bypass
+
+Purpose of the blower bypass is to prevent strain on blower when inhale pinch valve is closed. This is the version for 3/8"ID alpha build.
+
+## Purchasing source abbreviations
+
+* **A** = Ali Express
```
[the remark this answers is not in the document]
Comment (gracecanfield) [reply 4_Manufacturing_Design/Alpha_Build_Instructions/Blower-Bypass.md]: Removed extras.

File: 4_Manufacturing_Design/Alpha_Build_Instructions/pizza_blower_bypass/README.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
+# Blower Bypass
+
+The blower bypass is an optional "topping" that can be integrated into the [pizza build](../../Alpha_Build_Instructions/pizza_build.md).
+
+The purpose of the blower bypass is to prevent strain on blower when the inhale pinch valve is closed. This is the version for 3/8"ID [alpha build](../../Alpha_Build_Instructions).
```
[the remark this answers is not in the document]
Comment (gracecanfield) [reply 4_Manufacturing_Design/Alpha_Build_Instructions/pizza_blower_bypass/README.md:5]: Seems harmless (nothing but potentially helpful), but can remove.

File: 4_Manufacturing_Design/Alpha_Build_Instructions/pizza_blower_bypass/README.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+## BOM
+
+### Purchasing Source Key
+
+* **A** = Ali Express
```
[the remark this answers is not in the document]
Comment (gracecanfield) [reply 4_Manufacturing_Design/Alpha_Build_Instructions/pizza_blower_bypass/README.md]: Removed extras.

File: 4_Manufacturing_Design/Alpha_Build_Instructions/pizza_blower_bypass/README.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+![Assembled](bypass-assembled.jpg)
+
+Splice the assembled bypass into the pneumatic circuit such that:
+
+* One fork is right after the blower assembly.
+* The second fork is right at the outlet, after the exhale valve.
+
+Integrated into pizza build:
+
+![Pizza-Assembled](pizza-advanced.jpg)
```
[the remark this answers is not in the document]
Comment (gracecanfield) [reply 4_Manufacturing_Design/Alpha_Build_Instructions/pizza_blower_bypass/README.md:51]: Updated.

File: 4_Manufacturing_Design/Alpha_Build_Instructions/pizza_build.md (hunk 1 above)
Comment (gracecanfield) [reply 4_Manufacturing_Design/Alpha_Build_Instructions/pizza_build.md:143]: Added.

File: 4_Manufacturing_Design/Alpha_Build_Instructions/pizza_blower_bypass/README.md (hunk 2 above)
Comment (gracecanfield) [reply 4_Manufacturing_Design/Alpha_Build_Instructions/pizza_blower_bypass/README.md:38]: Updated.

**Pull request merged**